.github/workflows/release.yml

```diff
@@ -21,6 +22,26 @@ jobs:
           bump-minor-pre-major: true
           version-file: "judoscale-ruby/lib/judoscale/version.rb"
 
+  update-gemfile-versions:
+    name: Update version in Gemfile.lock files
+    needs: release-please
+    runs-on: ubuntu-latest
+    if: ${{ needs.release-please.outputs.pr_created == 'true' }}
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: "3.4"
+
+      - name: Update versions
+        run:
+          bin/foreach bundle install
+          git commit -a -m 'Update Gemfile.lock versions'
```

Comment:
Hopefully this will work, or it will be a bit of trial and error...

Reply:
Ah cool, I did not think of merging this to the release branch to test there.

I guess I could try both: merge to main first, then the bug fix, and see if it works... if it doesn't, then I can tweak it on the release branch to skip the condition and make it run on that branch as you suggested.